dev/io.openliberty.cdi.4.0.internal_fat/fat/src/io/openliberty/cdi40/internal/fat/config/LegacyConfigTest.java

```diff
@@ -42,8 +42,8 @@
 public class LegacyConfigTest extends FATServletClient {
     public static final String SERVER_NAME = "CDI40Server";
 
-    public static final String LEGACY_EMPTY_BEANS_APP_NAME = "LagacyEmptyBeans";
-    public static final String LEGACY_BEANS10_APP_NAME = "LagacyBeans10";
+    public static final String LEGACY_EMPTY_BEANS_APP_NAME = "LegacyEmptyBeans";
+    public static final String LEGACY_BEANS10_APP_NAME = "LegacyBeans10";
 
```

Comment:
Sort of unrelated to this change but further down in this file on line 84, you start the server with validation disabled, even though you expect all the deployed applications to start correctly.

Reply:
I consolidated the tests, hopefully they make more sense now